tox.ini

```diff
@@ -95,13 +92,12 @@ deps =
     py27: mock
     {env:_PYTEST_TOX_EXTRA_DEP:}
 distribute = true
-changedir=testing
 setenv =
     {[testenv]setenv}
     PYTHONDONTWRITEBYTECODE=1
 passenv = USER USERNAME TRAVIS
 commands =
-    {env:_PYTEST_TOX_COVERAGE_RUN:} pytest -n auto {posargs:.}
+    {env:_PYTEST_TOX_COVERAGE_RUN:} pytest -n auto
```

Comment:
Why are you removing `posargs` from some testenvs? I find them useful to pass extra arguments to envs.

Reply:
Yes, agreed - this was not intentional.

Reply:
It was just one though, fixed.